### PR TITLE
solution to the palindrome exercise

### PR DESCRIPTION
**Pull request by kengneruphine**
The solution to Issue #12

### CODE REVIEW

File: Week1/assignments/Ruphine-Kengne/palindrome.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//Compare the string to lowercase
+//We will split the string into characters, reverse the array of characters, join the characters back to form a string,
+//and then test the created string against what was originally received
+
+const isPalindrome = str => {
+    if (str.toLowerCase() === str.toLowerCase().split('').reverse().join('')) {
+        return true;
+    } else {
+        return false;
+    }
```
Comment (acha-bill) [inline Week1/assignments/Ruphine-Kengne/palindrome.js]: You could just do a 1 liner:
`return str.toLowerCase() === str.toLowerCase().split('').reverse().join('')`

**Comment by acha-bill** [comment]
First of all, your solution works! 👍 

Can you think of a way to improve it? In terms of time or space complexity?
You used 3 expensive operations, `split, reverse, join`. We can solve this problem without these.

File: Week1/assignments/Ruphine-Kengne/palindrome.js (hunk 1 above)
Comment (kengneruphine) [reply Week1/assignments/Ruphine-Kengne/palindrome.js]: Okay, thanks

**Comment by kengneruphine** [comment]
I will think of another way to implement it and update this PR with the solution

**Comment by kengneruphine** [comment]
Updated solution

**Pull request merged**